plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteMergeTableHandle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.trino.plugin.jdbc.JdbcColumnHandle;
+import io.trino.plugin.jdbc.JdbcMergeTableHandle;
+import io.trino.plugin.jdbc.JdbcTableHandle;
+
+public class IgniteMergeTableHandle
+        extends JdbcMergeTableHandle
```

Comment:
Why  plain `JdbcMergeTableHandle` is not good and you need to create `IgniteMergeTableHandle` class?

Reply:
`class io.trino.plugin.jdbc.JdbcOutputTableHandle cannot be cast to class io.trino.plugin.ignite.IgniteOutputTableHandle (io.trino.plugin.jdbc.JdbcOutputTableHandle and io.trino.plugin.ignite.IgniteOutputTableHandle are in unnamed module of loader 'app')`
There will be cast exception, I think it's because of the JSON de/serialization, so created a specified class carry `IgniteOutputTableHandle`.